packages/ui/src/components/ResourceBlock/ResourceBlock.tsx

```diff
@@ -0,0 +1,226 @@
+import { ReactElement, ReactNode, useId } from "react";
+import CheckCircleIcon from "@heroicons/react/20/solid/CheckCircleIcon";
+import ExclamationCircleIcon from "@heroicons/react/20/solid/ExclamationCircleIcon";
+import ArrowSmallRightIcon from "@heroicons/react/20/solid/ArrowSmallRightIcon";
+import { useIntl } from "react-intl";
+
+import { commonMessages } from "@gc-digital-talent/i18n";
+
+import Heading, { HeadingLevel } from "../Heading";
+import Link, { LinkProps } from "../Link";
+import { HydrogenAttributes } from "../../types";
+
+export const colorOptions = [
+  "primary",
+  "secondary",
+  "tertiary",
+  "quaternary",
+  "quinary",
+] as const;
+
+type CardColor = (typeof colorOptions)[number];
+
+const headingBarStyleMap: Record<CardColor, Record<string, string>> = {
+  primary: {
+    "data-h2-background-color": "base(primary.lightest)",
+    "data-h2-border-bottom-color": "base(primary.darkest)",
+  },
+  secondary: {
+    "data-h2-background-color": "base(secondary.lightest)",
+    "data-h2-border-bottom-color": "base(secondary.darkest)",
+  },
+  tertiary: {
+    "data-h2-background-color": "base(tertiary.lightest)",
+    "data-h2-border-bottom-color": "base(tertiary.darkest)",
+  },
+  quaternary: {
+    "data-h2-background-color": "base(quaternary.lightest)",
+    "data-h2-border-bottom-color": "base(quaternary.darkest)",
+  },
+  quinary: {
+    "data-h2-background-color": "base(quinary.lightest)",
+    "data-h2-border-bottom-color": "base(quinary.darkest)",
+  },
+};
+
+const wrapperStyleMap: Record<CardColor, Record<string, string>> = {
+  primary: {
+    "data-h2-color": "base(primary.darkest)",
+  },
+  secondary: {
+    "data-h2-color": "base(secondary.darkest)",
+  },
+  tertiary: {
+    "data-h2-color": "base(tertiary.darkest)",
+  },
+  quaternary: {
+    "data-h2-color": "base(quaternary.darkest)",
+  },
+  quinary: {
+    "data-h2-color": "base(quinary.darkest)",
+  },
+};
+
+interface RootProps {
+  title: ReactNode;
+  headingColor?: CardColor;
+  headingAs?: HeadingLevel;
+  children: ReactElement<ItemProps> | Array<ReactElement<ItemProps>>; // Restricts children to only expected items;
+}
+
+const Root = ({
+  title,
+  headingColor = "primary",
+  headingAs = "h3",
+  children,
+}: RootProps) => {
+  const headingId = useId();
+  return (
+    <div
+      data-h2-shadow="base(larger)"
+      data-h2-border-radius="base(rounded)"
+      data-h2-background-color="base(foreground)"
+      role="list"
+      aria-labelledby={headingId}
```

Comment:
![image](https://github.com/user-attachments/assets/5c030085-ab7a-4081-a7e3-71d38f344bbc)

Did you want to make this a list `li`? If so you'll need to add `role=listitem` to the children.

Reply:
Oh, good catch.  The children do have that role, further down.  I moved the list role down the tree a bit.
[move list role](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11200/commits/3a25f5f3605d7dc9585f3bdb991f310060847ac6)

Reply:
Do you need to move the aria-labelledby as well? :sweat_smile:

Reply:
OK, as discussed I left the list unlabelled and switched the top level element to a nav.
![image](https://github.com/user-attachments/assets/4a91ff8d-19a4-42d5-9e75-1278ad36defd)
[switch to nav element](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11200/commits/eb7a2a012e75ed74a06d2c7e8692de70d50bac42)